.travis.yml

```diff
@@ -135,18 +140,29 @@ script:
     # Insert build information into compiled binary using ldflags
     - CGO_ENABLED=0 go build -ldflags "-X main.Commit=`echo $TRAVIS_COMMIT` -X main.Tag=`echo $TRAVIS_TAG` -X main.Branch=`echo $TRAVIS_BRANCH` -X main.BuildNumber=`echo $TRAVIS_BUILD_NUMBER`"
 
-    # Start test environment
-    - bash ./tests/testenv.sh
+    # Build docker image from docker file
+    - sudo docker build -t $DOCKER_REPOSITORY .
 
-    # Only install acceptance testing dependencies after License checking is complete
-    - npm install validator chance jasmine-node frisby
+    # Clone integration repo. for api testing (purposely after license checking..)
+    - mv integration integration_old && git clone https://github.com/mendersoftware/integration.git
```

Comment:
The original `integration` directory is part of deployments' source code. Why not just `git clone https://github.com/mendersoftware/integration.git integration_new` ? In fact you, you are free to clone the integration repo elsewhere. Moving around/removing source code is a bit awkward.